app/cdap/components/RecipeManagement/RecipeForm/index.tsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { FormEvent, ChangeEvent } from 'react';
+import { FormControl } from '@material-ui/core';
+import T from 'i18n-react';
+import { IRecipeFormProps } from 'components/RecipeManagement/types';
+import {
+  FormFieldWrapper,
+  Label,
+  Form,
+  ErrorTextField,
+  NormalTextField,
+  StyledTextAreaAutosize,
+  CancelButton,
+  SaveButton,
+  ErrorLabel,
+  CreateRecipeFormButtonWrapper,
+  EditRecipeFormButtonWrapper,
+} from 'components/RecipeManagement/RecipeForm/styles';
+
+export const CREATE_RECIPE = 'createRecipe';
+
+export default function RecipeForm({
+  recipeFormData,
+  isRecipeNameError,
+  recipeNameErrorMessage,
+  onRecipeNameChange,
+  onFormSubmit,
+  setRecipeFormData,
+  onCancel,
+  isSaveDisabled,
+  recipeFormAction,
+}: IRecipeFormProps) {
+  const StyledLabel = isRecipeNameError ? ErrorLabel : Label;
+  const StyledTextField = isRecipeNameError ? ErrorTextField : NormalTextField;
+  const StyledFormButtonWrapper =
+    recipeFormAction === CREATE_RECIPE
+      ? CreateRecipeFormButtonWrapper
+      : EditRecipeFormButtonWrapper;
+
+  return (
+    <>
+      <Form
+        onSubmit={(event: FormEvent<HTMLFormElement>) => onFormSubmit(event)}
+        data-testid="recipe-form-parent"
+      >
+        <FormFieldWrapper>
+          <StyledLabel data-testid="recipe-name-label" component="span">
+            {recipeFormAction === CREATE_RECIPE
+              ? T.translate('features.WranglerNewUI.RecipeForm.labels.createRecipeNameLabel')
+              : T.translate('features.WranglerNewUI.RecipeForm.labels.editRecipeNameLabel')}
```

Comment:
You could use this way which avoids redundant usage.
```javascript
const PREFIX = 'features.WranglerNewUI.RecipeForm.labels'; 
{T.translate(`${PREFIX}.editRecipeNameLabel`)}
```

Reply:
Fixed, used PREFIX